tests/meta/orders/test_meta_orders_status.py

```diff
@@ -240,7 +240,7 @@ def test_store_microbial_samples_data_analysis_stored(
 
     # THEN it should store the samples
     assert base_store.samples().count() > 0
-    assert base_store.families().count() == 0
+    assert base_store.families().count() > 0
```

Comment:
Same here. Why was it 0 before and now one? Needs commenting

Reply:
fixed